variables.tf

```
@@ -426,6 +426,12 @@ variable "kafka_topic_name" {
   default     = ""
 }
 
+variable "s3_bucket_expiry" {
+  description = "Number of days after which apiary S3 bucket expire"
```

```suggestion
  description = "Number of days after which Apiary S3 buckets expire"
```
Does the bucket expire? Or do the objects within the bucket expire?

---

Agree.  Should be something like `Number of days after which objects in the Apiary S3 buckets expire`